docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM centos:centos6
```

Comment:
@jmcanterafonseca We still have 3 files, a README, a dockerfile, and a docker-compose. I leave it as is now, but if you feel it would be better at the root of the project I'll move it.

NTC?